corehq/apps/app_manager/id_strings.py

```diff
@@ -90,6 +90,16 @@ def detail_column_header_locale(module, detail_type, column):
     )
 
 
+@pattern('m%d.%s.%s_%s_%d.header')
+def invisible_column_header_locale(module, detail_type, column):
+    return u"m{module.id}.{detail_type}.{d.model}_{field}_{d_id}.header".format(
```

Comment:
I still think that unless this is producing a different string from `detail_column_header_locale` we should just use `detail_column_header_locale`